.github/synthesis/all.json

```diff
@@ -12,17 +12,17 @@
   {"top": "switchCalendar1k",              "stage": "hdl"},
   {"top": "switchCalendar1kReducedPins",   "stage": "pnr"},
 
-  {"top": "boardTestExtended",             "stage": "test",   "targets": "All"          },
-  {"top": "boardTestSimple",               "stage": "test",   "targets": "All"          },
-  {"top": "extraProbesTest",               "stage": "test",   "targets": "Specific [0]" },
-  {"top": "fincFdecTests",                 "stage": "test",   "targets": "Specific [-1]"},
-  {"top": "fullMeshHwCcTest",              "stage": "test",   "targets": "All"          },
-  {"top": "fullMeshHwCcWithRiscvTest",     "stage": "test",   "targets": "All"          },
-  {"top": "fullMeshSwCcTest",              "stage": "test",   "targets": "All"          },
-  {"top": "hwCcTopologyTest",              "stage": "test",   "targets": "All"          },
-  {"top": "linkConfigurationTest",         "stage": "test",   "targets": "All"          },
-  {"top": "syncInSyncOut",                 "stage": "test",   "targets": "All"          },
-  {"top": "temperatureMonitor",            "stage": "test",   "targets": "All"          },
-  {"top": "transceiversUpTest",            "stage": "test",   "targets": "All"          },
-  {"top": "vexRiscvTest",                  "stage": "test",   "targets": "Specific [-1]"}
+  {"top": "boardTestExtended",             "stage": "test"},
+  {"top": "boardTestSimple",               "stage": "test"},
+  {"top": "extraProbesTest",               "stage": "test"},
```

Comment:
```suggestion
```
`extraProbesTest` was removed (as it tested the now removed YAML configuration files), so CI should fail on this. I think it's a good idea to run all tests (not just those in `staging.json`) once more before merging.